eventstore/memory/memory.go

```diff
@@ -104,16 +80,24 @@ func (e *Memory) Get(ctx context.Context, id string, aggregateType string, after
 	return &iterator{events: events}, nil
 }
 
-// GlobalEvents will return count events in order globally from the start posistion
-func (e *Memory) GlobalEvents(start, count uint64) ([]core.Event, error) {
+// Close does nothing
```

Comment:
The fact that the function body is empty means that this comment is redundant.
```suggestion
```

Reply:
I close this as it has passed alot of water since the review and it was conducted when the PR was in a early stage.